app/V2/auth/models.py

```diff
@@ -76,4 +80,17 @@ def verify_hashed_password(password, hashed_password):
         """Method to verify password with the hashed password"""
         return pbkdf2_sha256.verify(password, hashed_password)
 
+    @classmethod
+    def find_by_nationaid(cls, id):
+        """This method gets a user using email"""
+        try:
+            cursor.execute("select * from users where nationalid = %s", (id,))
+            user = cursor.fetchone()
```

Comment:
Here the docstrings says that the `method gets a user using email` yet the implementation gets a user using national_id. Kindly correct that.

Reply:
Noted.I will correct this

Reply:
Noted.I will correct this.